dms/models/dms_file.py

```diff
@@ -560,6 +577,39 @@ def _inverse_content(self):
             for vals, ids in updates.items():
                 self.browse(ids).write(dict(vals))
 
+    def _create_model_attachment(self, vals):
+
+        if "default_directory_id" in self._context:
+            default_directory_id = self.env["dms.directory"].search(
+                [("id", "=", self._context["default_directory_id"])]
+            )
+
+            vals["directory_id"] = default_directory_id.id
+
+        directory_id = self.env["dms.directory"].search(
+            [("id", "=", vals["directory_id"])]
+        )
```

Comment:
Change is necessary to solved travis
```
        directory_id = self.env["dms.directory"].with_user(SUPERUSER_ID).search(
            [("id", "=", vals["directory_id"])]
        )
```

IMO this is a problem with _search function in directory that is not correct (isn't override and is needed) because not apply groups correctlly (not in this PR).